src/tools/Reporting/Reporting/Reporter.cs

```diff
@@ -108,7 +110,8 @@ public string GetJson()
                 build,
                 os,
                 run,
-                tests
+                tests,
+                criticalErrors = HasCriticalValidationErrors
```

Comment:
I added this line in here to say that if there are any critical validation errors, then they will be surfaced as a field "criticalErrors" in the run JSON. The value is true if there are critical errors and false if there aren't. I noticed that we have a few critical errors configured in the project such as having too many benchmarks in one category.